client/reader/stream/post.jsx

```diff
@@ -204,25 +205,8 @@ const Post = React.createClass( {
 
 			featuredSize = this.getFeaturedSize( maxWidth );
 		}
-
-		return useFeaturedEmbed
-			? <div
-					ref="featuredEmbed"
-					className="reader__post-featured-video"
-					key="featuredVideo"
-					dangerouslySetInnerHTML={ { __html: featuredEmbed.iframe } } />  //eslint-disable-line react/no-danger
-			: <div
-					className="reader__post-featured-image"
-					onClick={ this.handlePermalinkClick }>
```

Comment:
Yeah, looks like an old one.  I ended up at the initial commit trying to find where this might have been introduced.  Just wondering if we need it since we never had it.